src/patternfly/components/Check/examples/Check.md

```diff
@@ -8,23 +8,49 @@ cssPrefix: pf-c-check
 ```hbs title=Basic
 {{#> check}}
   {{#> check-input check-input--attribute='id="check1" name="exampleCheck1"'}}{{/check-input}}
-  {{#> check-label check-label--attribute='for="check1"'}}Check 1{{/check-label}}
+  {{#> check-label check-label--attribute='for="check1"'}}Check{{/check-label}}
 {{/check}}
+```
+
+```hbs title=Checked
+{{#> check}}
+  {{#> check-input check-input--attribute='id="check2" name="exampleCheck2" checked'}}{{/check-input}}
```

Comment:
A more sustainable way to manage the `id`, `name` and `for` attrs would be to use `check--id="check-checked"`, `check--id="check-label-wrapping-input"`, etc, then use that var for the values of `id`, `name`, and `for`. Just a nice to have, doesn't have to be part of this PR.